### DISCUSSION TITLE
Implementação do método every do JS

### DISCUSSION THREAD

**Discussion by JohnPetros** [DesignLiquido/delegua#665] [Geral]
O método [every](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/every)  do JS retorna `true` se cada iteração no vetor resultar em `true` e `false` se pelo menos uma iteração não conseguir resultar em `true`.

```
const isBelowThreshold = (currentValue) => currentValue < 40;

const array1 = [1, 30, 39, 29, 10, 13];

console.log(array1.every(isBelowThreshold)); // true
```

Como nome estava pensando em `paraTodos` em vez do mais obvio `todos`.

O que acham?

**Comment by leonelsanchesdasilva** [comment]
@JohnPetros Sim, acho que podemos implementar como sinônimo.

**Reply by leonelsanchesdasilva** [reply]
@JohnPetros Na verdade, o método se chama `todosEmCondicao`. Ainda acha necessário termos o sinônimo?

**Reply by JohnPetros** [reply]
Onde fica documentado `todosEmCondicao`, @leonelsanchesdasilva?

**Reply by leonelsanchesdasilva** [reply]
@JohnPetros Escrevi agora: https://github.com/DesignLiquido/delegua/wiki/Fun%C3%A7%C3%B5es-nativas#todosemcondicao
